bookkeeper-server/src/main/java/org/apache/bookkeeper/net/NetUtils.java

```diff
@@ -85,4 +92,46 @@ public static String resolveNetworkLocation(DNSToSwitchMapping dnsResolver, Inet
         return rNames.get(0);
     }
 
+    public static String getLocalHost() throws UnknownHostException {
+        String ip = InetAddress.getLocalHost().getHostAddress();
+        if (InetAddress.getLocalHost().isLoopbackAddress()) {
+            return getAddress();
+        } else {
+            return ip;
+        }
+
+    }
+
+    private static boolean isboolIp(String ipAddress) {
```

Comment:
please call this method **acceptIpAddress** or **isIpAddressValid** or **canUseIpAddress**
please make it visibility "package" and add unit tests.

from the regexp I see that we won't allow IPv6 addresses

Reply:
I will call this method isIpAddressValid